app/controllers/foodcoop/users_controller.rb

```diff
@@ -1,6 +1,18 @@
 class Foodcoop::UsersController < ApplicationController
   def index
-    @users = User.undeleted.natural_order
+    sort_param_map = {
```

Comment:
what about merging them with the `sort_param_map` in `app/controllers/admin/users_controller.rb` and move it to the model too?

Reply:
Done!